Documentation/ReleaseNotes/5.4.md

```diff
@@ -0,0 +1,30 @@
+# SwiftPM 5.4 Release Notes
+
+SwiftPM 5.4 includes several improvements, all of which are opt-in based on specifying a tools version of 5.4.
+
+
+### Executable targets
+
+Swift packages that specify a 5.4 tools version can now explicitly declare targets as executable, which allows the use of the `@main` keyword in package code.
+
+This is done by declaring the target using `executableTarget()` in the package manifest, telling SwiftPM that the target should be considered to be executable regardless of whether it contains a file with a base name of `main`.
+
+The compiler still applies its usual rules when compiling an executable module, so a single target cannot, for example, have a file named `main.swift` and another file that uses `@main`.
+
+### Package dependency caching
+
+Swift Package Manager caches now package dependency repositories on a per-user basis, which reduces the amount of network traffic and increases performance of dependency resolution for subsequent uses of the same package.
```

Comment:
for the record, it also caches compiled manifests

Reply:
Good point.  I've added this.